src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTrackingOptInRequestsOasis.java

```diff
@@ -20,326 +20,326 @@
+//  )
+//  public void adsTrackingRejectedForQualaroo(List<String> urlPatterns) {
+//    networkTrafficInterceptor.startIntercepting();
+//    TrackingOptInModal modal = new TrackingOptInModal();
+//    modal.rejectOptInModal(driver, POLAND, ADS_HOME_PAGE);
+//
+//    modal.verifyTrackingRequestsNotSend(urlPatterns, networkTrafficInterceptor);
+//  }
+//
+//  @NetworkTrafficDump(useMITM = true)
+//  @Execute(trackingOptIn = false)
+//  @Test(
+//      dataProviderClass = TrackingOptInDataProvider.class,
+//      dataProvider = "adsKikimoraAcceptedDataProvider",
+//      groups = "AdsOptInAcceptedOasis"
+//  )
+//  public void verifyAdsTrackingAcceptedForKikimora(String[] instantGlobals,
+//                                                   List<String> urlPatterns) {
+//    networkTrafficInterceptor.startIntercepting();
+//    TrackingOptInModal modal = new TrackingOptInModal();
+//    modal.acceptOptInModal(driver, POLAND, ADS_HOME_PAGE, instantGlobals);
+//
+//    modal.verifyTrackingRequestsSend(urlPatterns, networkTrafficInterceptor);
+//  }
+//
+//  @NetworkTrafficDump(useMITM = true)
+//  @Execute(trackingOptIn = false)
+//  @Test(
+//      dataProviderClass = TrackingOptInDataProvider.class,
+//      dataProvider = "adsKruxDataProvider",
+//      groups = "AdsOptInAcceptedOasis"
+//  )
+//  public void adsTrackingAcceptedForKrux(String[] instantGlobals, List<String> urlPatterns) {
+//    networkTrafficInterceptor.startIntercepting();
+//    TrackingOptInModal modal = new TrackingOptInModal();
+//    modal.acceptOptInModal(driver, POLAND, ADS_HOME_PAGE, instantGlobals);
+//
+//    modal.verifyTrackingRequestsSend(urlPatterns, networkTrafficInterceptor);
+//  }
+//
+//  @NetworkTrafficDump(useMITM = true)
+//  @Execute(trackingOptIn = false)
+//  @Test(
+//      dataProviderClass = TrackingOptInDataProvider.class,
+//      dataProvider = "adsNetzAthletenDataProvider",
+//      groups = "AdsOptInAcceptedOasis"
+//  )
+//  public void adsTrackingAcceptedForNetzAthleten(String[] instantGlobals,
+//                                                 List<String> urlPatterns) {
+//    networkTrafficInterceptor.startIntercepting();
+//    TrackingOptInModal modal = new TrackingOptInModal();
+//    modal.acceptOptInModal(driver, GERMANY, ADS_HOME_PAGE, instantGlobals);
+//
+//    modal.verifyTrackingRequestsSend(urlPatterns, networkTrafficInterceptor);
+//  }
+//
+//  @NetworkTrafficDump(useMITM = true)
+//  @Execute(trackingOptIn = false)
+//  @Test(
+//      dataProviderClass = TrackingOptInDataProvider.class,
+//      dataProvider = "adsNordicsDataProvider",
+//      groups = "AdsOptInAcceptedOasis"
+//  )
+//  public void adsTrackingAcceptedForNordics(List<String> urlPatterns) {
+//    networkTrafficInterceptor.startIntercepting();
+//    TrackingOptInModal modal = new TrackingOptInModal();
+//    modal.acceptOptInModal(driver, DENMARK, ADS_HOME_PAGE);
+//
+//    modal.verifyTrackingRequestsSend(urlPatterns, networkTrafficInterceptor);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)